### DISCUSSION TITLE
Enable Experiments

### DISCUSSION THREAD

**Discussion by esarella** [itzg/docker-minecraft-bedrock-server#274] [General]
Can we enable experiments by using params in server.properties or docker-compose.yml, If so what are the params?

**Comment by esarella** [comment]
I want to enable holiday creator feature and Molang Features

**Comment by itzg** [comment]
From a quick search it sounds like those are attributes of the world save data. Creating that first in the client and then using that world data seems to be suggested approach.

**Reply by pezcutler** [reply]
I've done that and interestingly it then doesn't show up in the lan for console.  can connect from Add Server but this prevents my kids connecting from ps

**Reply by itzg** [reply]
@pezcutler that will be an unrelated problem the bedrock server itself has with LAN discovery. If you search the issues there has been many a discussion about this mystery. Unfortunately I haven't heard of any consistent solution.

**Comment by josephdpurcell** [comment]
See https://github.com/itzg/docker-minecraft-bedrock-server/discussions/373#discussioncomment-10210491 for details on enabling experiments.
